Cartfile

```diff
@@ -1,4 +1,4 @@
-github "mxcl/PromiseKit" ~> 4.4
-github "jdg/MBProgressHUD" ~> 1.0.0
-github "JSONModel/JSONModel" "master"
-github "Boris-Em/BEMCheckBox" ~> 1.4.1
+github "Boris-Em/BEMCheckBox" "1.4.1"
+github "JSONModel/JSONModel" "1.7.0"
+github "jdg/MBProgressHUD" "1.1.0"
+github "mxcl/PromiseKit" "6.2.8"
```

Comment:
@falnatsheh any reason to pin to exact versions here?

Reply:
The reason I used these specific values because it matches with what we used with CocoaPod. This way the dependencies stay consistent between CocoaPod and Carthage.

Reply:
Hmm - looks like our Cocoapods is set to `~>` which will incorporate updates as per:
```
'~> 0.1.2' Version 0.1.2 and the versions up to 0.2, not including 0.2 and higher
'~> 0.1' Version 0.1 and the versions up to 1.0, not including 1.0 and higher
'~> 0' Version 0 and higher, this is basically the same as not having it.
```

Reply:
Thanks for pointing that out, it looks like Carthage has similar operator (from their [documentations](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement)): `~> 1.0 for “compatible with version 1.0”`. So this could be updated to: 

```
github "Boris-Em/BEMCheckBox" ~> 1.4.1
github "JSONModel/JSONModel" ~> 1.7.0
github "jdg/MBProgressHUD" ~> 1.0.0
github "mxcl/PromiseKit" ~> 6.2.8
```